release notes/v0.32.0.md

```diff
@@ -0,0 +1,107 @@
+
+
+### Considerable performance improvements for arrival rate executors ([#1955](https://github.com/k6io/k6/pull/1955))
+
+Due to a *wrong* re-usage for a particular internal data structure, the arrival rate executors were having a much worse performance than expected. With this release they should be a lot more performant, especially with large numbers of VUs.
+
+
+### Updating the majority of our dependencies and dropping some
+
+The list is too long and we have been postponing updating for quite some time now, but we have finally updated all our dependencies that we don't want to drop. This could lead to some stability problems, which is why it was done early on in the cycle. While the team has not found any regressions, given all the updates we could have missed something so please [open a issue](https://github.com/k6io/k6/issues) if you find anything.
+
+Some notable updates:
+- goja, the JS engine we use got support for `let`/`const` which let us disable a Babel plugin. Previous to this if you had a particularly *long* script sometimes Babel took upwards of 30 minutes to transpile. Now even our worst contender that previously took 51 minutes is transpiled in less than a minute :tada:. Also `globalThis` is now available.
+- updating the gRPC libraries fixed bug [#1928](https://github.com/k6io/k6/issues/1928) and probably others. [#1937](https://github.com/k6io/k6/pull/1937)
+
+### `ArrayBuffer` is now supported in all JS APIs dealing with binary data, including in WebSocket messages ([#1841](https://github.com/k6io/k6/pull/1841))
+
+Besides the minor breaking changes (see the "Breaking changes" section below), it's now possible to send binary WS messages with the `socket.sendBinary()` function and to receive binary messages with the `binaryMessage` event handler:
+
+  ```javascript
+  const binFile = open('./file.pdf', 'b');
+
+  export default function () {
+    ws.connect('http://wshost/', function(socket) {
+      socket.on('open', function() {
+        socket.sendBinary(binFile);
+      });
+
+      socket.on('binaryMessage', function(msg) {
+        // msg is an ArrayBuffer, so we can wrap it in a typed array directly.
+        new Uint8Array(msg);
+      });
+    });
+  }
+  ```
+
+### Official releases arm64 for MacOS and Linux ([#2000](https://github.com/k6io/k6/pull/2000))
+
+We will now publish binary arm64 releases for MacOS and Linux for new k6 releases. Support for these new architectures should be stable, given Go's cross-platform compatibility, but please report any issues you experience.
+
+## Other enhancements and UX improvements
+
+- Options: k6 will now warn you on unrecognised configuration JS options in most cases instead of just silently ignoring them. ([#1919](https://github.com/k6io/k6/pull/1919))
+- error_code: the tag `error_code` should now be set more accurately in some cases. [#1952](https://github.com/k6io/k6/pull/1951)
```

Comment:
It's a silly thing, but we're not consistent with the usage of parenthesis around PR references. @MStoykov would you mind adding them here and everywhere else they're missing?